docs/docs/workflows/create.md

```diff
@@ -174,12 +174,71 @@ terraform apply
 The Constellation [apply step](#the-apply-step) requires the already created `constellation-config.yaml` and the `constellation-state.yaml`.
 Create the `constellation-state.yaml` using the output from the Terraform state and the `constellation-conf.yaml`:
 
+<tabs groupId="provider-output">
+<tabItem value="aws-output" label="AWS">
+
+```bash
+yq eval '.version ="v1"' --inplace constellation-state.yaml
+yq eval ".infrastructure.initSecret =\"$(terraform output initSecret | jq -r | tr -d '\n' | hexdump -ve '/1 "%02x"' && echo '')\"" --inplace constellation-state.yaml
+yq eval ".infrastructure.clusterEndpoint =\"$(terraform output out_of_cluster_endpoint | jq -r)\"" --inplace constellation-state.yaml
+yq eval ".infrastructure.inClusterEndpoint =\"$(terraform output in_cluster_endpoint | jq -r)\"" --inplace constellation-state.yaml
+yq eval ".infrastructure.ipCidrNode =\"$(terraform output ip_cidr_nodes | jq -r)\"" --inplace constellation-state.yaml
+yq eval ".infrastructure.uid =\"$(terraform output uid | jq -r)\"" --inplace constellation-state.yaml
+yq eval ".infrastructure.name =\"$(terraform output name | jq -r)\"" --inplace constellation-state.yaml
+yq eval ".infrastructure.apiServerCertSANs =$(terraform output -json api_server_cert_sans)" --inplace constellation-state.yaml
+```
+
+</tabItem>
+<tabItem value="azure-output" label="Azure">
+
+:::info
+
+  If the enforcement policy is set to `MAAFallback` in `constellation-config.yaml`, a manual update to the MAA provider's policy is necessary.
+  The update can be applied with the following commands, where `<VERSION>` is the version of Constellation that should be set up. (e.g. `v2.12.0`)
+
+  ```bash
+  https://github.com/edgelesssys/constellation/tree/<VERSION>
+  cd constellation/hack/maa-patch
+  go run . $(terraform output attestationURL | jq -r)
```

Comment:
Azure CLI doesn't support this operation, does it?

Reply:
Theoretically, it does, but it would require a fairly long script that we would have to potentially maintain along the code in our CLI, why I opted to share the code and create a tool instead.